docs/MapColonies/DEM/Services/elevation/README.md

```diff
@@ -22,13 +22,22 @@ Allows users to request elevation(height) information at a given location(s)/pos
 
 ## Terminology
 
-Elevation info might be returned from following materials:
+### Product types
+
+Elevation info might be returned from following material types:
 1. DTM - terrain materials.
 2. DSM - surface materials.
-3. MIXED - according to most detailed(best resolution) materials.
+3. MIXED - according to most detailed (best resolution) materials.
 
-## Usage
+### Confidence level
+
+Calculated for each product and is represented by two values:
+* `resolutionMeter` - precision on the horizontal plane (may be interpolated), meaning that for every `x` meters there is a value in the mesh
+* `absoluteAccuracyLEP90` - each height measurement has a 90% chance to have a deviation of at most `x` meters from the given value (between `height` - `x` and `height` + `x`)
+
+It is extremely important to provide all of the information about the product which the height was extracted from, for each point you query, in order to allow the user to make the best decision based on the data.
```

Comment:
GOOD, but think that part "in order to allow the user to make the best decision based on the data" is redundant

Reply:
Removed, but added an `info` banner for the required values to be displayed.